File: online-judge-solutions/Codeforces/595A.cpp
//https://codeforces.com/problemset/problem/595/A
//Lang		:	GNU C++14
//Time		:	31 ms
//Memory	:	100 KB
#include<iostream>
using namespace std;
int main(){
	int n,m,r=0,s=0,x;cin>>n>>m;
	int v[n*2*m];
	for(int i=0;i<n*2*m;i++){
		cin>>v[i];
		r+=v[i];
		if(i%2!=0 && r>0){s++;r=0;}
	}
	cout<<s<<endl;
	return 0;
}

File: online-judge-solutions/Codeforces/595A.py
#https://codeforces.com/problemset/problem/595/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
n,m=map(int,input().split())
v=[x for i in range(n) for x in map(int,input().split())]
print(sum([1 for i in range(0,len(v),2) if sum(v[i:i+2])>0]))
